module/plugins/captcha/ReCaptcha.py

```diff
@@ -354,3 +356,45 @@ def _challenge_v2(self, key, secure_token=None):
             self.fail(_("Recaptcha max retries exceeded"))
 
         return result, challenge
+
+    # solve interactive captcha (javascript required), use when non-JS captcha fallback for v2 is not allowed
+    def _challenge_v2_js(self, key):
```

Comment:
remove `key` from the parameters as it is not used, because the key is retrieved later on.